src/main/java/org/sagebionetworks/bridge/sdk/models/upload/UploadFieldDefinition.java

```diff
@@ -70,40 +121,113 @@ public UploadFieldType getType() {
-        return result;
-    }
-
-    @Override
-    public boolean equals(Object obj) {
-        if (this == obj) {
+    public boolean equals(Object o) {
+        if (this == o) {
             return true;
         }
-        if (obj == null || getClass() != obj.getClass()) {
+        if (o == null || getClass() != o.getClass()) {
             return false;
         }
-        UploadFieldDefinition other = (UploadFieldDefinition) obj;
-        return Objects.equals(name, other.name) && Objects.equals(required, other.required)
-                && Objects.equals(type, other.type);
+        UploadFieldDefinition that = (UploadFieldDefinition) o;
+        return required == that.required &&
+                Objects.equals(fileExtension, that.fileExtension) &&
+                Objects.equals(mimeType, that.mimeType) &&
+                Objects.equals(minAppVersion, that.minAppVersion) &&
+                Objects.equals(maxAppVersion, that.maxAppVersion) &&
+                Objects.equals(maxLength, that.maxLength) &&
+                Objects.equals(name, that.name) &&
+                type == that.type;
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public int hashCode() {
+        return Objects.hash(fileExtension, mimeType, minAppVersion, maxAppVersion, maxLength, name, required, type);
+    }
+
+    /** {@inheritDoc} */
     @Override
     public String toString() {
-        return String.format("UploadFieldDefinition[name=%s, required=%b, type=%s]", name, required, type.name());
+        return "UploadFieldDefinition[" +
+                "fileExtension=" + fileExtension +
+                ", mimeType=" + mimeType +
+                ", minAppVersion=" + minAppVersion +
+                ", maxAppVersion=" + maxAppVersion +
+                ", maxLength=" + maxLength +
+                ", name=" + name +
+                ", required=" + required +
+                ", type=" + type.name() +
+                "]";
     }
 
     /** Builder for UploadFieldDefinition */
     public static class Builder {
+        private String fileExtension;
+        private String mimeType;
+        private Integer minAppVersion;
+        private Integer maxAppVersion;
+        private Integer maxLength;
         private String name;
         private Boolean required;
         private UploadFieldType type;
 
+        /** @see org.sagebionetworks.bridge.sdk.models.upload.UploadFieldDefinition#getFileExtension */
+        public String getFileExtension() {
+            return fileExtension;
+        }
+
+        /** @see org.sagebionetworks.bridge.sdk.models.upload.UploadFieldDefinition#getFileExtension */
+        public Builder withFileExtension(String fileExtension) {
+            this.fileExtension = fileExtension;
+            return this;
+        }
+
+        /** @see org.sagebionetworks.bridge.sdk.models.upload.UploadFieldDefinition#getMimeType */
+        public String getMimeType() {
+            return mimeType;
+        }
```

Comment:
I have no idea. There were getters before, so I added them for consistency. I'll check to see if they're being used, and if not, I'll delete them.